_tests_/port.test.js

```diff
@@ -0,0 +1,30 @@
+const Port = require("../src/Port.js");
+
+describe("Port", () => {
+  const port = new Port("Dunkirk");
+  it("Has a name", () => {
+    expect(port.name).toBe("Dunkirk");
+  });
+  it("Constructs a new instance of a port", () => {
+    expect(new Port()).toBeInstanceOf(Port);
+  });
```

Comment:
Couple of things here, the other tests have the first letter of the test name in lower case so you should use the same format here. Also, its generally better to create a new instance to test inside each test, so you should move the creation pf the `port` variable inside the test, and as always leave a clear line between each test:
```
  it("has a name", () => {
    const port = new Port("Dunkirk");
    expect(port.name).toBe("Dunkirk");
  });

  it("constructs a new instance of a port", () => {
    expect(new Port()).toBeInstanceOf(Port);
  });
```